### DISCUSSION TITLE
Strange behaviour

### DISCUSSION THREAD

**Discussion by shingoo79** [Thyraz/Sonos-Kids-Controller#29] [Q&A]
Hi guys,

I am observing that my Sonos Kids Controller sometimes behaves very strangely. The following observations I have made:

* The configured albums (from Spotify) are always displayed correctly.
* You can also browse everything properly.
* Sometimes everything works properly.
* Sometimes the album is not sent correctly to Sonos, which means that play and pause only stops and starts the currently running track. But that is not the selected album on the controller.

Does anyone have any idea what could be going on here?

Greetings
Roman

**Comment by Thyraz** [comment]
I noticed this problem too, often after I streamed something from my mobile phone or Spotify Connect to the Sonos speaker.
A reboot normally fixes the problem.

There has been an update for node-sonos-http-api a few days ago. The description looks like the fix might be related to that problem.
So maybe try to update node-sonos-http-api ...

**Reply by shingoo79** [reply]
Thanks for the reply.

I updated the node-sonos-http-api. It worked for a couple of hours. Now it is again very strange.

The audio pauses irregularly then it starts suddenly again. Sometimes it jumps back some chapters. Sometimes it jumps to the first chapter. And at the moment it is again the same as in my previous post.

Restarting the raspi does not help...

**Comment by shingoo79** [comment]
It seems to work much more stable using localhost instead of the IP in the configuration files. Perhaps this helps others, too.
